code/modules/vehicles/armored/__armored.dm

```diff
@@ -671,8 +672,7 @@
 	var/image/secondary_overlay
 
 /atom/movable/vis_obj/turret_overlay/Destroy()
-	if(primary_overlay)
-		QDEL_NULL(primary_overlay)
+	QDEL_NULL(primary_overlay)
```

Comment:
Doesn't this mean runtimes due to null passes to qdel is possible

Reply:
nulls don't cause runtimes in qdel